quantecon/tests/test_dle.py

```diff
@@ -35,7 +35,7 @@ def setUp(self):
         deltah = np.array([[.9]])
         thetah = np.array([[1]]) - deltah
         ub = np.array([[30, 0, 0]])
-        x0 = np.array([[5, 150,1,0,0]]).T
+        # x0 = np.array([[5, 150, 1, 0, 0]]).T  # unused
```

Comment:
@natashawatkins would you have time to check this?

Reply:
it's not used anywhere else in this file, but I can't figure out why it's there in the first place